lua/lspconfig/server_configurations/remark_language_server.lua

```diff
@@ -0,0 +1,51 @@
+local util = require 'lspconfig.util'
+
+local bin_name = 'remark-language-server'
+local cmd = { bin_name, '--stdio' }
+
+if vim.fn.has 'win32' == 1 then
+  cmd = { 'cmd.exe', '/C', bin_name, '--stdio' }
+end
+
+return {
+  default_config = {
+    cmd = cmd,
+    filetypes = { 'markdown' },
+    root_dir = util.find_git_ancestor,
+    single_file_support = true,
+  },
+  docs = {
+    package_json = 'https://raw.githubusercontent.com/remarkjs/remark-language-server/main/package.json',
```

Comment:
This package.json doesn't contain any relevant information. These are  only used for generating`settings` and `initOptions`